src/app/ubs/ubs-admin/components/ubs-admin-customers/ubs-admin-customer-details/ubs-admin-customer-details.component.ts

```diff
@@ -20,4 +32,31 @@ export class UbsAdminCustomerDetailsComponent implements OnInit {
     this.localStorageService.removeCurrentCustomer();
     this.location.back();
   }
+
+  openPopUp(column: string, data: string, userId: string): void {
+    this.dialogConfig.disableClose = true;
+    const modalRef = this.dialog.open(CommentPopUpComponent, this.dialogConfig);
+
+    this.translate.get(column).subscribe((data) => (modalRef.componentInstance.header = data));
```

Comment:
The translate.get subscription within modalRef.afterClosed creates a nested subscription, which can lead to memory leaks or unintended behavior if not unsubscribed.